services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -182,7 +186,7 @@ private void handleCommand(CommandMessage message, String replyToTopic, Destinat
         } else if (data instanceof DumpPortDescriptionRequest) {
             doDumpPortDescriptionRequest(message, replyToTopic, replyDestination);
         } else {
-            logger.error("unknown data type: {}", data.toString());
+            logger.error("Unknown data type: {}", data);
```

Comment:
Lets change it to "Unable to handle {data.className()} request - handler not found".